docs/src/routes/docs/components/dropdown-shell.svx

```diff
@@ -0,0 +1,98 @@
+---
+name: DropdownShell
+relatedComponents:
+  - name: Dropdown
+    link: ./docs/components/dropdown
+---
+
+<script>
+  import { DropdownShell, Dropdown, Button } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import ShadowPreview from 'src/components/docs/shadow-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+</script>
+
+# DropdownShell
+
+The container for the [`Dropdown`](./docs/components/dropdown) activator and content.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <DropdownShell let:toggle>
+      <Button on:click={toggle}>
+        Can contain more than one
+      </Button>
+      <Dropdown>
+        <div class="padded">
+          first
+        </div>
+      </Dropdown>
+      <Dropdown right>
+        <div class="padded">
+          second
+        </div>
+      </Dropdown>
+    </DropdownShell>
+    <DropdownShell>
+      Useless without a <code>&lt;Dropdown /&gt;</code>
+    </DropdownShell>
+  </div>
+  <div slot="source">
+
+```svelte
+<DropdownShell let:toggle>
+  <Button on:click={toggle}>
+    Can contain more than one
```

Comment:
Oh god, no, let's not endorse that. This is uninteded behaviour and I see 0 logical reasons for it to be used

Reply:
Just wanted to show different content than in the Dropdown and got a bit creative)